internal/helper/loadbalancermachine.go

```diff
@@ -487,14 +487,19 @@ func generatePromtailConfig(
 		return "", err
 	}
 
+	proxy := ""
+	if logForward.ProxyURL != "" {
+		proxy = "\n    proxy_url: '" + logForward.ProxyURL + "'"
+	}
+
 	return `server:
   disable: true
 
 positions:
   filename: /tmp/positions.yaml
 
 clients:
-  - url: '` + logForward.LokiURL + `'
+  - url: '` + logForward.LokiURL + "'" + proxy + `
```

Comment:
Can we do this differently? This is extremely hard to process.

Reply:
I've cleaned the line up a bit. Is that better or do you have some more drastic change in mind?